model/task_lifecycle.go

```diff
@@ -639,6 +639,14 @@ func doBisectStepback(ctx context.Context, t *task.Task) error {
 		return errors.Wrapf(err, "setting stepback info for task '%s'", nextTask.Id)
 	}
 
+	grip.Debug(message.Fields{
```

Comment:
Is this is a temporary log? If so, is there a ticket to remove it eventually?

Reply:
There's no ticket in the epic or that I can find that is to remove the logging, there also hasn't been any discussion to remove it

Reply:
I guess the question is more about what we will plan to do with the log. If this is a thing just to verify that the algorithm is correct before the project is finished, then it seems like we would want to eventually remove it. But if we have an ongoing purpose to keep it around, then we should probably upgrade it to a info log so we know that it's not a stray debug log that nobody uses.

Reply:
I talked to Malik about it and it's more so an ongoing purpose, so I'll update it to an info log

Reply:
Cool, I already LGTM'd so feel free to merge whenever!